plugins/chat/assets/javascripts/discourse/routes/chat-channel-thread.js

```diff
@@ -0,0 +1,31 @@
+import DiscourseRoute from "discourse/routes/discourse";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+import { schedule } from "@ember/runloop";
+
+export default class ChatChannelThread extends DiscourseRoute {
```

Comment:
Can I please get some feedback on this to make sure it's not doing anything silly? I don't create routes often